lib/rubocop/cop/lint/boolean_symbol.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for boolean symbols.
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #   :true
+      #
+      # @example
+      #
+      #   # good
+      #
+      #   true
+      #
+      class BooleanSymbol < Cop
+        MSG = 'Use `%<name>s` instead of `:%<name>s`.'.freeze
```

Comment:
I think the cop description and the message should allude to the fact that this might be accidental, since there are actual cases where you'd want `:true` or `:false`, and just saying "use `true` instead" is actually wrong. WDYT? 🙂